public-api/src/main/java/org/pmiops/workbench/google/GoogleAnalyticsService.java

```diff
@@ -0,0 +1,11 @@
+package org.pmiops.workbench.google;
+
+import java.io.IOException;
+
+public interface GoogleAnalyticsService {
+
+    int trackEventToGoogleAnalytics(
+            String cid, String category, String action, String label, String value)
+            throws NullPointerException, IOException;
```

Comment:
opt_nit: looks a bit unusual to me to declare an unchecked exception (NPE); I understand where this came from (Preconditions) but I might leave it off all the same.

Reply:
Done